sparse/etc/pulse/ia32_rt5659_soundcard.pa

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/pulseaudio -nF
+#
+# This file is part of PulseAudio.
+#
+# PulseAudio is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# PulseAudio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with PulseAudio; if not, write to the Free Software Foundation,
+# Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.
+
+# This startup script is used only if PulseAudio is started per-user
+# (i.e. not in system mode)
+
+.nofail
+
+.fail
+
+load-module module-droid-keepalive
+
+load-module module-meego-parameters cache=1 directory=/var/lib/nemo-pulseaudio-parameters use_voice=false
+
+### Automatically restore the volume of streams
+load-module module-stream-restore-nemo restore_device=no restore_volume=yes restore_muted=no route_table=/etc/pulse/x-maemo-route.table fallback_table=/etc/pulse/x-maemo-stream-restore.table use_voice=false sink_volume_table=/etc/pulse/x-maemo-sink-volume.table
+
+load-module module-match table=/etc/pulse/x-maemo-match.table key=application.name
+
+### Automatically augment property information from .desktop files
+### stored in /usr/share/application
+#load-module module-augment-properties
+
+load-module module-null-sink sink_name=sink.null rate=48000
+
+load-module module-alsa-card device_id=2 card_name=card.rt5659 sink_name=sink.rt5659 source_name=source.rt5659 rate=48000
+
+load-module module-null-sink sink_name=sink.fake.sco rate=8000 channels=1
+load-module module-null-source source_name=source.fake.sco rate=8000 channels=1
+load-module module-bluez4-discover sco_sink=sink.fake.sco sco_source=source.fake.sco
```

Comment:
This line needs to be
load-module module-bluetooth-discover bluez4_args="sco_sink=sink.fake.sco sco_source=source.fake.sco"